apps/src/StudioApp.js

```diff
@@ -831,23 +821,28 @@ StudioApp.prototype.showInstructions_ = function(level, autoClose) {
   // If there is an instructions block on the screen, we want the instructions dialog to
   // shrink down to that instructions block when it's dismissed.
   // We then want to flash the instructions block.
-  var hideFn = null;
   var hideOptions = null;
   var endTargetSelector = "#bubble";
 
   if ($(endTargetSelector).length) {
     hideOptions = {};
     hideOptions.endTarget = endTargetSelector;
+  }
 
+  var hideFn = _.bind(function() {
```

Comment:
Any particular reason to use `_.bind` here instead of [`Function.prototype.bind`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind)?